src/containers/ServiceGraphPageContainer.ts

```diff
@@ -7,9 +7,15 @@ import ServiceGraphPage from '../pages/ServiceGraph/ServiceGraphPage';
 import { ServiceGraphDataActions } from '../actions/ServiceGraphDataActions';
 
 const mapStateToProps = (state: KialiAppState) => ({
-  graphTimestamp: state.serviceGraphDataState.timestamp,
-  graphData: state.serviceGraphDataState.graphData,
-  isLoading: state.serviceGraphDataState.isLoading
+  graphTimestamp: state.serviceGraph.dataState.timestamp,
+  graphData: state.serviceGraph.dataState.graphData,
+  isLoading: state.serviceGraph.dataState.isLoading,
+  summaryData: state.serviceGraph.selectedObject
+    ? {
+        summaryTarget: state.serviceGraph.selectedObject.obj,
+        summaryType: state.serviceGraph.selectedObject.type
```

Comment:
Is "type" a language keyword? I think we should consider renaming this - "type" just seems too "keyword-y". I dunno if this is a problem or not.

Reply:
Yes, `type` is a typescript keyword.